qemu/tests/qemu_img_bitmap.py

```diff
@@ -108,6 +118,8 @@ def _check_bitmap_add(img, bitmap_name):
         res = _map_nbd_bitmap(nbd_server, nbd_port, nbd_export_bitmap)
         match_info = {"start": 0, "length": 1048576, "depth": 0,
                       "present": False, "zero": False, "data": False}
+        if host_qemu in VersionInterval('[8.2.0,)'):
+            match_info["compressed"] = False
```

Comment:
I am curious if we need to cover compressed = True, it looks we never care about that from your code change

Reply:
Talked with the feature owner, we don't cover the compressed images in the two cases, so compressed is always set to True, we have other test cases which cover the compressed images' testing.